Data collection guidance (#1061)
open-contracting/standard: docs/guidance/build/data_collection_tools.md

Consider using the form if:

* You don’t have the capacity to develop or install a software tool
* Your users have access to reliable internet connections
* Your users are unfamiliar with spreadsheets
* Data entry will be done by many different users
* You want to minimize the work required to collate data
* You don’t need to update the data about a contracting process after it is first entered

The form includes a subset of fields from the **tender** and **buyer** sections of OCDS. The OCDS Helpdesk can help you extend and adapt the form to suit your needs.

Read more about using and customizing the form in the [resource guide](https://www.open-contracting.org/resources/ocds-data-collection-form/). The OCDS helpdesk can also provide guidance on using the form to collect data and help you to analyze the data that you collect.

## Data collection spreadsheet

The [data collection spreadsheet](https://www.open-contracting.org/resources/data-collection-spreadsheet/) is available in Google Sheets and for offline use in Microsoft Excel.

> Data is entered directly into the template, which structures and formats it to conform to OCDS.

Reviewer: Unless I'm mistaken:

```suggestion
Data is entered directly into the template, which is structured and formatted to conform to OCDS.
```

Data entered the template can be checked and converted using the [OCDS Data Review Tool](https://standard.open-contracting.org/review/) and published in either spreadsheet or JSON format.

Consider using the spreadsheet template to collect data if:

* You don’t have the capacity to develop or install a software tool
* You need to collect data without access to an internet connection
* Your users are familiar with using spreadsheets
* Data entry will be done by a small number of users
* You have the capacity to collate data entered in multiple spreadsheets

> * You need to make multiple updates over the life of a contracting process

Reviewer: Consider adding **emphasis** around some key words that distinguish each list from the other lists, e.g. **without** access, are **familiar**, by a **small number of**, etc.

Author: Done in https://github.com/open-contracting/standard/pull/1061/commits/9aa990a192ed4e8bc1135bd7e12c24fb0b1693b0, feel free to suggest changes

## Contrataciones Abiertas tool

[Contrataciones Abiertas](https://github.com/INAImexico/Contrataciones_abiertas_v2) is a tool developed by the Mexican government and it is made up of two main modules:

* Information capture module
* Visualization module

The objective of the tool is to enable the monitoring of contracting processes by using and producing OCDS data.

Consider using Contrataciones Abiertas if:

* You have the resources and technical capacity to [set up the tool](https://github.com/INAImexico/Contrataciones_abiertas_v2/raw/master/Manual%20de%20instalaci%C3%B3n.docx)
* Data entry will be done by different users
* You want to validate your data before publication
* You want to minimize the work required to collate and publish data
* You want a [dashboard and visualizations](https://www.open-contracting.org/2020/06/12/mexicos-inai-launches-new-open-source-tool-to-upload-and-visualize-open-contracting-data/) of your data
* Spanish is your (and your users') main language

> The tool is open source and is documented in Spanish [here](https://github.com/INAImexico/Contrataciones_abiertas_v2). You can also find an introduction to the tool in [our blog](https://www.open-contracting.org/2020/06/12/mexicos-inai-launches-new-open-source-tool-to-upload-and-visualize-open-contracting-data/).

Reviewer: Never use "here" links.

```suggestion
The tool is open source and is [documented in Spanish](https://github.com/INAImexico/Contrataciones_abiertas_v2). You can also find an introduction to the tool in [our blog](https://www.open-contracting.org/2020/06/12/mexicos-inai-launches-new-open-source-tool-to-upload-and-visualize-open-contracting-data/).
```

## What next?

> However you choose to collect and structure your data, you should publish it online using an [open license](../../publish/#license-your-data) so that others can make use of it.
> 
> Consider how you will make it easy for users to discover the data you have published. For example, by publishing it on an existing procurement portal, on your organization’s website, or an open data portal.

Reviewer: Leave the licensing content out. It's covered in the publish step.

```suggestion
However you choose to collect and structure your data, consider how you will make it easy for users to discover the data you have published. For example, by publishing it on an existing procurement portal, on your organization’s website, or an open data portal.
```

Author: Replaced with a link to the relevant guidance in https://github.com/open-contracting/standard/pull/1061/commits/004745d3dd8fec737cc28690a4bf8f64c45ebdce
